decidim-blogs/app/cells/decidim/blogs/post_g_cell.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Blogs
+    # This cell renders the Grid (:g) post card
+    # for an given instance of a Post
+    class PostGCell < Decidim::CardGCell
+      private
+
+      def photo
```

Comment:
Is okay to use the method, but you could use 
```
delegate :photo, to: :model
```

Reply:
Changed